site/_includes/macros/landing-section.njk

```diff
@@ -24,18 +24,18 @@
               <div class="item">
                 <div class="item__inner height-full display-flex direction-column">
                   <div class="item__title">
-                    <a href="{{ post.url }}">{{ post.title or post.data.title }}</a>
+                    <a href="{{ post.url }}">{{ post.title or post.data.title | md | safe }} </a>
                   </div>
-                  <p class="desc flex-1">{{ post.description or post.data.description }}</p>
+                  <div class="desc flex-1">{{ post.description or post.data.description | md | safe }}</div>
                 </div>
               </div>
             </div>
           {% else %}
             <div class="item height-full">
               <div class="item__inner
                 height-full">
-                <div class="item__title"><a href="{{ post.url }}">{{ post.title or post.data.title }}</a></div>
-                <p class="desc flex-1">{{ post.description or post.data.description }}</p>
```

Comment:
Why change from p to div?

Reply:
I brought back the p tag. I accidentally changed it while updating this section